automatic/minishift/update.ps1

```diff
@@ -13,28 +13,31 @@ function global:au_SearchReplace {
     ".\legal\VERIFICATION.txt" = @{
       "(?i)(^\s*location on\:?\s*)\<.*\>" = "`${1}<$($Latest.ReleaseURL)>"
       "(?i)(^\s*software.*)\<.*\>"        = "`${1}<$($Latest.URL64)>"
-      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType32)"
+      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType64)"
       "(?i)(^\s*checksum\:).*"            = "`${1} $($Latest.Checksum64)"
     }
 
     "$($Latest.PackageName).nuspec" = @{
-      "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseURL)`${2}"
+      "(\<copyright\>.*?-)\d{4}(.*?\</copyright\>)" = "`${1}$($Latest.ReleaseYear)`${2}"
```

Comment:
As with the other package, can you guarantee that this will ensure that the copyright matches the copyright specified by the Software Authors?

If not, then this should be a manual process instead of an automatic one.